test/testng/org/openjdk/jextract/test/toolprovider/TestClassGeneration.java

```diff
@@ -115,8 +115,8 @@ public static Object[][] structMembers() {
             { "Foo", C_LONG_LONG.withName("ll"), long.class,          10L },
             { "Foo", C_FLOAT.withName("f"),     float.class,         10F },
             { "Foo", C_DOUBLE.withName("d"),    double.class,        10D },
-            { "Bar", C_INT.withName("a"),       int.class,           10 },
-            { "Bar", C_INT.withName("b"),       int.class,           10 },
+            { "Bar$0", C_INT.withName("a"),       int.class,           10 },
```

Comment:
This change is needed because the test looks for a private field (the OFFSET field), and such field is declared in the typedef class' superclass. The old logic was generating things backwards, so we didn't have this issue.